app/src/main/java/com/example/tasky/agenda/data/network/event/mappers/EventMappers.kt

```diff
@@ -27,7 +29,37 @@ fun EventDto.toEvent(): Event {
     )
 }
 
-fun AttendeeDto.toAttendee(hostId: String): Attendee {
+fun Event.toCreateEventRequest(): CreateEventRequest {
+    return CreateEventRequest(
+        id = id,
+        title = title,
+        description = description,
+        timeFrom = timeFrom.toString(),
```

Comment:
Should the request payload use `from` and `to` as the keys here?  I am curious if its working so far  😅